lib/ansible/module_utils/common/text/converters.py

```diff
@@ -237,7 +309,7 @@ def to_text(obj, encoding='utf-8', errors=None, nonstring='simplerepr'):
 to_native = to_text
 
 
-def jsonify(data, **kwargs):
+def jsonify(data: _t.Any, **kwargs) -> str:
```

Comment:
For kwargs, let's use
```suggestion
def jsonify(data: object, **kwargs: object) -> str:
```
if possible.

Reply:
I think it's a bad idea to add types that are wrong. Acceptable keyword arguments like `skipkeys`, `ensure_ascii`, `check_circular`, ... accept values of specific types, and not of type `object`. I think in such situations it's better to not add types so places like this which need more work are easier to find.

Reply:
It's best to use a `TypedDict` or better mechanisms for `kwargs`. But `object` is almost always better than `Any`. Everything in Python inherits from `object`. So it's always correct (unless attributes/interfaces that are present in more specific subtypes are accessed).

https://mypy.readthedocs.io/en/stable/dynamic_typing.html#any-vs-object